templates/answerfile_4.4_basic.txt.j2

```diff
@@ -1 +1,5 @@
 {% include "./templates/answerfile_4.3_basic.txt.j2" %}
+
+{% if ovirt_engine_grafana_enable is defined %}
+QUESTION/1/OVESETUP_GRAFANA_ENABLE=str:{{ ovirt_engine_grafana_enable | ternary('yes','no')  }}
```

Comment:
Please use the same answer syntax as other options.
```
OVESETUP_GRAFANA_CORE/enable=bool:True
OVESETUP_GRAFANA_CORE/grafanaDbCreatedByUs=bool:True
```
You can check /etc/ovirt-engine-setup.conf.d/ on engine.